### ISSUE TITLE
Split multiple collections better

### ISSUE THREAD

**Issue by alan-knight** [dart-lang/dart_style#218]
The API takes a List of maps (keyframes) and a map for timing information. The formatter is putting this all on one line, which is pretty hard to follow. It seems like the second argument should at least be split out onto its own line.

      var body = document.body;
      var fontSize = body.getComputedStyle().fontSize;
      var player = body.animate(
          [{"font-size": "500px"}, {"font-size": fontSize}], {"duration": 100});

**Comment by munificent** [comment]
This is kind of dense, but it fits in one line which is why the formatter is happy with it.

I can't think of any heuristic for when it should add splits for something that doesn't need any at all, so I think I'm just going to leave this behavior as-is.
